moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js

```diff
@@ -73,13 +82,29 @@ const useColumns = ({
           return user ? `${user.first_name} ${user.last_name}` : "";
         },
         renderEditCell: (props) => {
+          // the team member object for the current row
+          const currentRowMember =
+            data?.moped_project_by_pk?.moped_proj_personnel.find(
+              (user) => user.project_personnel_id === props.id
+            );
+          // filter out existing team members from list of options unless they are the current row member
+          // that way the current member remains an option when editing a row
+          const teamMembersWithoutDuplicates = data?.moped_users.filter(
```

Comment:
I'm not 100% sold on this variable name. Because our existing list of users already does not have duplicates in it. This is a list of team members that aren't already on the project, so its *preventing* duplicates. 

maybe `unassignedTeamMembers`?

Reply:
cool i like that, just updated!